### ISSUE TITLE
Promote small pools with a better ranking system

### ISSUE THREAD

**Issue by Frastap0** [input-output-hk/daedalus#2569]
Would it be possible to address the issue regarding how small pools are listed. There is little or no chance of securing new delegators with the current ranking system as we are coloured red, have potential rewards of 0ADA and have no effective rank.

As small pools help increase the security and decentralisation of the Cardano network, I think it is important to properly promote them on the staking section of Daedalus.

**Comment by nikolaglumac** [comment]
@Frastap0 we will get back to you next week. Cheers!

**Comment by MMA-stakepool** [comment]
The red colouring and potential rewards = 0.00000 ADA are very easy fixes: 
    - Colouring pools with smaller stake red gives the impression that they are very bad/don't work. 
    - 0.00000 ADA potential rewards is not even true, as there are potential rewards for any stake as long as the pool functions. This sends the message "this pool doesn't work at all" which is usually not the case. 

A bigger overhaul of the ranking system could also be considered, with many ideas floating around but of course this requires more research and work. These two basic fixes, colour and 0 potential reward, would go a long way to help smaller pools get started with very little research/coding effort.

**Comment by StakeWithPride** [comment]
Perhaps we can replace the rankings view with a newer one that promotes decentralization. 

We see this in game server selection UIs like World of Warcraft where players are encouraged to join less utilized servers. 

Or in Kiva.org, the default choices are featured, recommended, almost funded, etc. Imagine if the default Kiva order was ranked by best ROI; fewer would get micro-loans. 

Let's have a uniform experience across Daedalus and Yoroi. Daedalus needs an overhaul. Yoroi needs to remove ADAPools; too many biases built into that one too. 

One idea is setting default criteria like Single Pools, under N delegation, with N% ROI etc, and then let users play with sliders.

**Comment by rafek47** [comment]
The current ranking in Daedalus is actually helping pool farms & pools with higher levels of saturation and giving no chance to the small hard working SPOs. It does not make sense to list the pools like that in a truly decentralized Peer to Peer system. If all the peers are equal, some of them shouldn't be marked as "garbage".

Adding some kind of randomness to the ranking would be very nice.
Having a choice to list pools by an Alliance also - Mission Driven, Bare metal etc.

Would be nice to start at least with the marking mentioned above, because this is really discouraging delegators.

**Comment by now0clock** [comment]
Thanks for pointing this out. As a newcomer to the platform, I was researching which wallet to use, how to stake, etc. After reading this issue, I did more research and discovered how diverse and ecclectic the choices of pools to stake with are.

I could see how something like red text and 'potential rewards "0.00000 ADA"' would turn people away, give them the wrong impression and might see that pool as "bad" even after the font color is changed. I even found trouble visiting a few ADA pool related domains that are on DNS blocklists. For instance, adapools.org & pooltool.io are on 'Energized Basic' block list. "pegasus-pool.firebaseio.com" showed up on another block list.

I am pretty cultured, have an open mind, and welcome new ideas and concepts. I am also pretty knowledgable about cryptocurrencies and the risks and potential scamming that it attracts. Unfortunately most people coming into this realm are not. In general I think most people suffer from xenophobia and can't think outside of their socio-economic culture-clashing ideals fed to them by mass media, and probably can't do proper research because X pool domain is on some adware/malware blocklist maybe up to the ISP level.

Apologies for my lengthy comment, but I thought I should add my 2 ADA's here. Thanks.

p.s. ADA is at $1.60USD right now.. :)

**Comment by nikolaglumac** [comment]
Thank you all for the extensive feedback and ideas you shared! 
The current ranking system in Daedalus depends on the rankings implementation on the cardano-node/cardano-wallet side. The way potential rewards and the ranks are calculated is solely up to the Node and Adrestia teams.
Of course, we can and we will improve the presentation and we count on your support 🙏 
I can not promise you that we will find a way to make everybody happy but I can promise you we will try!

**Comment by LeoniePhiline** [comment]
Are there corresponding bug reports in the Node and Adrestia repos?

**Comment by nikolaglumac** [comment]
> Are there corresponding bug reports in the Node and Adrestia repos?

I am not sure - please check and open a ticket if not 🙏 
PS. Make sure to link them.

**Comment by StakeWithPride** [comment]
Sharing someone else's idea: Instead of ranking by best to worst, what about using the reverse slot leader algorithm, to promote the pools that have less delegation? You could filter on ROI and other parameters to perhaps default to the 25th thru 75th percentiles most of the time, and 0-25, 75-100 percentiles some of the time.

**Comment by Frastap0** [comment]
Good morning, do we have any update from the devs? Thank you

**Comment by nikolaglumac** [comment]
There are many discussions being held with many development teams. This is not just a question of Daedalus. Will post an update once we have something to share.
Thanks for the patience 🙏

**Comment by bemnet4u** [comment]
Small pool operator here...I for sure think that there are possible ways to improve this. Just a way to mark mission driven pools and single pool operators might help increase decentralization. 

Please let us know the plans for this.

**Comment by darko-mijic** [comment]
Work on improving the ranking system is in progress. We should have updates on this soon. 

We are also working on a completely re-thought ranking system in parallel. 

Mission-driven pools can work together and operate a SMASH server with a curated list of stake pools. We would be happy to include such SMASH servers in Daedalus settings.

![image](https://user-images.githubusercontent.com/4280521/123910496-ce946000-d97a-11eb-95cf-39adaeda3f68.png)

**Comment by cybershaman73** [comment]
> Work on improving the ranking system is in progress. We should have updates on this soon.
> 
> We are also working on a completely re-thought ranking system in parallel.
> 
> Mission-driven pools can work together and operate a SMASH server with a curated list of stake pools. We would be happy to include such SMASH servers in Daedalus settings.
> 
> ![image](https://user-images.githubusercontent.com/4280521/123910496-ce946000-d97a-11eb-95cf-39adaeda3f68.png)

That's great news Darko, thank you. Is the process for submitting an MDP SMASH server already in place?

**Comment by robinboening** [comment]
@darko-mijic interesting idea. I am very curious if you discussed the on boarding of SMASH servers. Even that needs to be curated / controlled to some extend, otherwise you could quickly end up with a list of 100 servers, or even 1000 if some funny person deploys a fleet of SMASH docker containers.

**Comment by LeoniePhiline** [comment]
Ada holders could vote to curate the list.

**Comment by rp5210** [comment]
> Work on improving the ranking system is in progress. We should have updates on this soon.
> 
> We are also working on a completely re-thought ranking system in parallel.
> 
> Mission-driven pools can work together and operate a SMASH server with a curated list of stake pools. We would be happy to include such SMASH servers in Daedalus settings.
> 
> ![image](https://user-images.githubusercontent.com/4280521/123910496-ce946000-d97a-11eb-95cf-39adaeda3f68.png)

I have three SMASH servers running on mainnet, one for each of three different pool alliances.  After updating one to the new version of dbsync v12 the one for the CMDP (mission driven pools) is currently down for maintenance.  If you want to test the other two, one is for the SPA (single pool alliance - https://smash.panl.org) and the other for the xSPO alliance (https://xspo.panl.org).

Would greatly appreciate any feedback.

**Comment by rafek47** [comment]
I believe the majority of the people, especially the new ones in Cardano will never switch to different SMASH servers. 

Daedalus keeps showing 0 potential rewards in red color for most of the pools (even the ones regularly minting blocks and paying to delegates).
338 of the top 500 pools shown in Daedalus are Multipool operators. 

The chance new delegates end up in multitool farm is about 70%.

My comment is related to the ongoing Single VS Multitool discussion and a slow direction of re-centralization of Cardano.
I do believe rankings are playing a crucial role in that and they have the potential to reverse this trend. 

The question is, why do we build and design a decentralized system if we end up with about 20 entities minting 50% of all blocks?

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
